MekHQ/src/mekhq/gui/view/PersonViewPanel.java

```diff
@@ -587,369 +666,304 @@ private void fillStats() {
             gridBagConstraints.gridx = 1;
             gridBagConstraints.gridy = firsty;
-            gridBagConstraints.weightx = 0.5;
+            gridBagConstraints.weightx = 1.0;
             gridBagConstraints.insets = new Insets(0, 10, 0, 0);
             gridBagConstraints.fill = GridBagConstraints.NONE;
             gridBagConstraints.anchor = GridBagConstraints.NORTHWEST;
-            pnlStats.add(lblDuedate2, gridBagConstraints);
+            pnlFamily.add(lblSpouse2, gridBagConstraints);
+            firsty++;
         }
 
-        int secondy = 0;
+        if (campaign.getCampaignOptions().useParentage() && person.hasChildren()) {
+            lblChildren1.setName("lblChildren1"); // NOI18N //$NON-NLS-1$
+            lblChildren1.setText(resourceMap.getString("lblChildren1.text")); //$NON-NLS-1$
+            gridBagConstraints = new GridBagConstraints();
+            gridBagConstraints.gridx = 0;
+            gridBagConstraints.gridy = firsty;
+            gridBagConstraints.fill = GridBagConstraints.NONE;
+            gridBagConstraints.anchor = GridBagConstraints.NORTHWEST;
+            pnlFamily.add(lblChildren1, gridBagConstraints);
+
+            lblChildren2.setName("lblChildren2"); // NOI18N //$NON-NLS-1$
+            lblChildren2.setText(person.getChildList());
+            gridBagConstraints = new GridBagConstraints();
+            gridBagConstraints.gridx = 1;
+            gridBagConstraints.gridy = firsty;
+            gridBagConstraints.weightx = 1.0;
+            gridBagConstraints.insets = new Insets(0, 10, 0, 0);
+            gridBagConstraints.fill = GridBagConstraints.NONE;
+            gridBagConstraints.anchor = GridBagConstraints.NORTHWEST;
+            pnlFamily.add(lblChildren2, gridBagConstraints);
+            firsty++;
+        }        
+    }
+    
+    private void fillSkills() {
+    	
+    	//skill panel
+        pnlSkills = new JPanel(new GridBagLayout());
+        pnlSkills.setBorder(BorderFactory.createTitledBorder(resourceMap.getString("pnlSkills.title")));
+        pnlSkills.setBackground(Color.WHITE);
+             
+        //abilities and implants
+        lblAbility1 = new JLabel();
+        lblAbility2 = new JLabel();
+        lblImplants1 = new JLabel();
+        lblImplants2 = new JLabel();
+        lblTough1 = new JLabel();
+        lblTough2 = new JLabel();
+        lblEdge1 = new JLabel();
+        lblEdge2 = new JLabel();
+        lblEdgeAvail1 = new JLabel();
+        lblEdgeAvail2 = new JLabel();
+        
+        GridBagConstraints gridBagConstraints;
+        
         JLabel lblName;
         JLabel lblValue;
-
-        for(int i = 0; i < SkillType.getSkillList().length; i++) {
+        
+        int firsty=0;
+        int colBreak = Math.max((int) Math.ceil(person.getSkillNumber() / 2.0)+1, 3);
+        int addition = 0;
+        double weight = 0.5;
+        
+        int j = 0;
+        for(int i = 0; i < SkillType.getSkillList().length; i++) {      	
```

Comment:
Mixed tabs and spaces within this for loop

Reply:
fixed in commit 9a200a7b1